src/main/java/frc/robot/RobotContainer.java

```diff
@@ -55,6 +56,7 @@ public class RobotContainer {
   private final static Limelight subLimelight = new Limelight();
 
   private final Trigger gamePieceTrigger = new Trigger(() -> subTransfer.getGamePieceCollected());
+  private final Trigger readyToShoot = new Trigger(() -> subDrivetrain.isDrivetrainFacingSpeaker());
```

Comment:
We should also check for if rpm and pivot angle are correct

Reply:
I'll make a separate issue asap